docs/debugging.md

```diff
@@ -0,0 +1,192 @@
+---
+id: debugging_tips
+title: Tips and tricks for debugging & testing
+---
+
+This document provides some tips & utilities for Signatory developers.
+
+## Request a public key for an address from the remote signer
+
+```bash title="Using curl to "
+$ curl -XPOST \
+    -H 'Content-Type: application/json'\
+    -d '"131395aa01961837e74d50c9ce304c83e3baa60061f956d9c8703df1d6374e86417b3b753315000c000353cf00000000c18896cfe7d09893bc25997f24b46515e1e6c58a0deaba0a5f17dc0dc59ae038"' \
+    localhost:6732/keys/tz1M3kiQUMPcTseo72i1twdiV4iTY5yiGNSz
+{"signature":"edsigtcnGArmYo9cSLnV3cVfd3Kmgm3pbEDpTG2DMwNPBCFzgtV19KToyhVeu3vNX99HXqAsFnC6oDigzkvEbYJpA6e9gieQAav"}
+```
+
+## Reset a ledger watermark 
+
+The leger Tezos Baker app tracks a watermark and round value to prevent the double signing of operations. If you attempt to sign a block, endorsement or pre_endoresment more than once, the remote signer will return an error. 
+
+```json title="Sample error from octez tezos-signer. HTTP code is 500"
+[{"kind":"permanent","id":"signer.ledger","ledger-error":"Application level error (sign-with-hash): Incorrect data"}]
+```
+
+TODO: Include an example error from signatory 
```

Comment:
Should this to-do be converted into a task and removed from here?

Reply:
resolved